front-end/src/ui-components/CreationLabel.tsx

```diff
@@ -11,18 +11,19 @@ import Avatar from './Avatar';
 import InlineTag from './InlineTag';
 
 interface Props{
+	address?: string
```

Comment:
All these changes (additional props + additional conditions) in `CreationLabel` are never used by any component in this PR. We should revert them unless I missed something.